### PR TITLE
Fix ES search results size to 10000

### PR DESCRIPTION
**Pull request by infojunkie**

### CODE REVIEW

**Comment by infojunkie** [comment]
@caiosba plz review

File: lib/elasticsearch_relation.rb [hunk 1]
```diff
@@ -20,7 +19,7 @@ def all
   end
 
   def count
-    Annotation.search(@params.merge({ size: 0 })).total
+    Annotation.search(@params).total
```
Comment (caiosba) [inline lib/elasticsearch_relation.rb]: If we have more than 10.000 the `total` will be wrong... I think you shouldn't change this line... or is there a reason I'm not seeing?

File: lib/elasticsearch_relation.rb [hunk 2]
```diff
@@ -1,12 +1,11 @@
 # This class tries to reproduce an ActiveRecord relation
 class ElasticsearchRelation
   def initialize(params = {})
-    @params = params
+    @params = params.merge({ size: 10000 })
```
Comment (caiosba) [inline lib/elasticsearch_relation.rb]: I suggest the opposite: `{ size: 10000 }.merge(params)`, this way we can change it when calling

Comment (infojunkie) [inline lib/elasticsearch_relation.rb]: thanks

File: lib/elasticsearch_relation.rb (hunk 1 above)
Comment (infojunkie) [inline lib/elasticsearch_relation.rb]: I tested that the number set in `size` has no effect on the `total` result. E.g. I set `size` to 10 and it still returned a correct total of 14.

Comment (caiosba) [inline lib/elasticsearch_relation.rb]: Crazy, but thanks

**Pull request merged**